app/api/helpers/tasks.py

```diff
@@ -97,6 +97,22 @@ def resize_event_images_task(self, event_id, original_image_url):
         logging.exception('Error encountered while generating resized images for event with id: {}'.format(event_id))
 
 
+@celery.task(base=RequestContextTask, name='resize.user.images', bind=True)
+def resize_user_avatar_images_task(self, user_id, avatar_image_url):
+    user = safe_query(db, User, 'id', user_id, 'user_id')
+    try:
+        logging.info('User avatar image resizing tasks started {}'.format(avatar_image_url))
+        uploaded_images = create_save_image_sizes(avatar_image_url, 'user-image', user.id)
+        user.avatar_url = uploaded_images['avatar_url']
```

Comment:
This should be set to `original_image_url`

My oversite on that part

Reply:
```user.avatar_url``` ?